src/contracts/proxies/ODSafeManager.sol

```diff
@@ -303,12 +305,12 @@ contract ODSafeManager is IODSafeManager, Authorizable, Modifiable {
 
   /// @inheritdoc Modifiable
   function _modifyParameters(bytes32 _param, bytes memory _data) internal override {
-    address _address = abi.decode(_data, (address));
+    address _address = _data.toAddress();
```

Comment:
Why was this changed?

Reply:
_data.toAddress() was consistent with the rest of the contracts. and the toAddress() and .toBytes() functions seemed to work more consistently in testing than just abi.decode();  I had a few instances where when switching from decode to toBytes it fixed some issues.  didn't have a problem on the safeManager but I wanted to keep it consistent with the rest of the contracts.